dev/com.ibm.ws.jaxrs.2.0.server/src/com/ibm/ws/jaxrs20/server/LibertyJaxRsServerFactoryBean.java

```diff
@@ -71,8 +72,8 @@ public class LibertyJaxRsServerFactoryBean extends JAXRSServerFactoryBean {
 
     private final static TraceComponent tc = Tr.register(LibertyJaxRsServerFactoryBean.class);
 
-    private final List<JaxRsFactoryBeanCustomizer> beanCustomizers = new LinkedList<JaxRsFactoryBeanCustomizer>();
-    private Map<String, Object> beanCustomizerContexts;
+    private final CopyOnWriteArrayList<JaxRsFactoryBeanCustomizer> beanCustomizers = new CopyOnWriteArrayList<JaxRsFactoryBeanCustomizer>();
+    private ConcurrentHashMap<String, Object> beanCustomizerContexts;
```

Comment:
```suggestion
    private final List<JaxRsFactoryBeanCustomizer> beanCustomizers = new CopyOnWriteArrayList<JaxRsFactoryBeanCustomizer>();
    private Map<String, Object> beanCustomizerContexts;
```

Can still use the interface for the field type.